src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -134,6 +157,16 @@ void OverlayWindow::disable(bool trace_event)
         target_state->exit();
         target_state.reset();
         winkey_popup.reset();
+
+        bool success = UnhookWindowsHookEx(hook_handle);
+        if (success)
+        {
+            hook_handle = nullptr;
+        }
+        else
+        {
+            throw std::runtime_error("Cannot uninstall keyboard listener");

```

Comment:
Same as above, but in this case reporting the error to the user may be pointless.